apps/fulfillment/vite.config.ts

```diff
@@ -40,26 +40,10 @@ export default defineConfig({
         orientation: 'portrait-primary',
         icons: [
           {
-            src: '/icons/android-chrome-192x192.png',
-            sizes: '192x192',
-            type: 'image/png',
-          },
-          {
-            src: '/icons/android-chrome-512x512.png',
-            sizes: '512x512',
-            type: 'image/png',
-          },
-          {
-            purpose: 'maskable',
-            src: '/icons/android-chrome-maskable-192x192.png',
-            sizes: '192x192',
-            type: 'image/png',
-          },
-          {
-            purpose: 'maskable',
-            src: '/icons/android-chrome-maskable-512x512.png',
-            sizes: '512x512',
-            type: 'image/png',
+            src: '/icons/logo.svg',
+            sizes: '192x192 512x512',
```

Comment:
But what about png for chrome?

Reply:
Why do we need PNG for chrome?

Reply:
Maybe I got it wrong, but I think you said during the daily svg is not working for android (not chrome, sorry).